From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Tue, 31 Aug 2021 14:58:23 -0700
Subject: [PATCH] Permit unhandled configure options

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 configure | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure b/configure
index 0a4a0fc..ec2aa39 100755
--- a/configure
+++ b/configure
@@ -518,7 +518,8 @@ else
 			"")
 				break ;;
 			*)
-				usage 1 ;;
+				echo "WARNING: unhandled configure flag: $1"
+				shift ;;
 		esac
 	done
 fi
